_posts/izayak/2016-01-31-Ruotong-Clicky-Turtle.md

```diff
@@ -0,0 +1,14 @@
+---
+author: izayak
+title: "Ruotong's Clicky Turtle"
+layout: post
+---
+
+<iframe src="https://trinket.io/embed/python/cdde424311" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
Very creative- let's talk in class (maybe with your partner) about your helper function to see if we implement it.